azure-pipelines.yml

```diff
@@ -44,6 +44,28 @@ resources:
     image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
 
 stages:
+- stage: Create_BAR_ID_Tag
+  displayName: Create BAR ID Tag
+  jobs:
+  - template: /eng/common/templates/job/job.yml
+    parameters:
+      name: Create_BAR_ID_Tag
+      displayName: Create BAR ID Tag
+      timeoutInMinutes: 240
```

Comment:
Does this need to be so high?

Reply:
Hah! No. Copypasta... what do we think... 10 mins sufficient?

Reply:
I'd just take it out and let it be the azdo default (which is an hr)